CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## 1.3.1 - 2021-04-16
+- Update to infrastructure-bundle 2.4.1
+
```

Comment:
is this actually a new version? The real change is already reflected in https://github.com/newrelic/nri-ecs/releases/tag/v1.3.0

Reply:
I know, but we will need to release again with a new tag, to upload the manifest, no?

Reply:
Good point. I guess we could overwrite the tag but that would be ugly.